test/e2e/scripts/local-deploy.sh

```
@@ -0,0 +1,1548 @@
+  trap 'kill $_PF_PID 2>/dev/null || true' EXIT
+  sleep 3
+
+  # API key
+  API_KEY=$(kubectl exec -n "$MAAS_NAMESPACE" deployment/maas-api -- curl -sk \
+    "https://localhost:8443/v1/api-keys" \
+    -H "X-MaaS-Username: validate-user" \
+    -H 'X-MaaS-Group: ["system:authenticated"]' \
+    -H "Content-Type: application/json" \
+    -d '{"name":"validate"}' 2>/dev/null | jq -r '.key // empty')
+
+  _check "API key creation" '[[ -n "$API_KEY" ]] && [[ "$API_KEY" == sk-oai-* ]]'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)